### DISCUSSION TITLE
multi reference

### DISCUSSION THREAD

**Discussion by ayoubelmhamdi** [typst/typst#1458] [Q&A]
is there any way to use multi-reference in one document, like:

- INTRODUCTION
  - REFERENCE OF INTRODUCTION
- CH1
  - REFERENCE OF CH1
- CH2
  - REFERENCE OF CH2
- CONCLUSION
  - REFERENCE OF  CONCLUSION

**Comment by laurmaedje** [comment]
Closing in favor of https://github.com/typst/typst/issues/1472
